cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/task/dnszone/AzureDnsZoneCreationPoller.java

```diff
@@ -0,0 +1,43 @@
+package com.sequenceiq.cloudbreak.cloud.azure.task.dnszone;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.cloud.azure.task.AzurePollTaskFactory;
+import com.sequenceiq.cloudbreak.cloud.context.AuthenticatedContext;
+import com.sequenceiq.cloudbreak.cloud.exception.CloudConnectorException;
+import com.sequenceiq.cloudbreak.cloud.scheduler.SyncPollingScheduler;
+import com.sequenceiq.cloudbreak.cloud.task.PollTask;
+
+@Component
+public class AzureDnsZoneCreationPoller {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AzureDnsZoneCreationPoller.class);
+
+    private static final int DNS_ZONE_CREATION_CHECKING_INTERVAL = 1000;
+
+    private static final int DNS_ZONE_CREATION_CHECKING_MAX_ATTEMPT = 100;
+
+    private static final int MAX_FAILURE_TOLERANT = 1;
```

Comment:
these should be `@Values` so we could easily override them

Reply:
Good idea, worth doing for all the affected Pollers